widgetastic_manageiq/__init__.py

```diff
@@ -3106,12 +3106,12 @@ class chart(TimelinesChart):  # NOQA
 
     @property
     def is_displayed(self):
-        return self.title.text == "Timelines"
+        return self.title.text == "Timeline View"
 
     @property
     def is_timelines(self):
         """method to check title text for base Timelines without overriding is_displayed"""
-        return self.title.text == "Timelines"
+        return self.title.text == "Timeline View"
```

Comment:
same change backported to 5.10? if it changed in specific z-stream I'd use VersionPicker again.

Reply:
Yep, I believe that this is was just wrong and the `is_displayed` logic that uses this check got messed up as well.

Reply:
Actually this is due to BZ https://bugzilla.redhat.com/show_bug.cgi?id=1732517, I modified this method to workaround that BZ